modules/mono/editor/GodotTools/GodotTools/Build/MSBuildPanel.cs

```diff
@@ -122,7 +122,7 @@ public override void _Ready()
         {
             base._Ready();
 
-            CustomMinimumSize = new Vector2(0, 228) * EditorScale;
+            CustomMinimumSize = new Vector2i(0, 228 * EditorScale);
```

Comment:
Interesting GitHub feature for C# code. Why can't they do that for C++? :)
![image](https://user-images.githubusercontent.com/4701338/189141235-1940fcd7-d5d6-4aa3-b6ef-0519b68755d3.png)

Anyway if C# wants to be super explicit I guess you might do:
```
new Vector2i(0, (int)(228.0 * EditorScale))
```

Or maybe @godotengine/dotnet have a better suggestion.

Reply:
About the GitHub feature, I'm pretty sure the dotnet action we use in CI is what reports those errors so it should be possible to do it for C++ if someone implemented it (see the `core.error` and the `core.warning` commands in https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions).

---

About the `CustomMinimumSize`, I don't really have a better suggestion other than maybe:

```csharp
new Vector2i(0, 228 * (int)EditorScale);
```

Reply:
That one would be a problem as we want `228 * 1.5` to give `342`, not `228 * 1 = 228`.

Reply:
You are right, I don't know where my head was. By the way you are missing the parenthesis around `int` in your latest changes.